f8a_worker/graphutils.py

```diff
@@ -223,6 +223,12 @@ def create_package_dict(graph_results, alt_dict=None):
             }
             if epv['pkg'].get('cooccurrence_probability'):
                 pkg_dict['cooccurrence_probability'] = epv['pkg']['cooccurrence_probability']
+
+            # Add the co-occurence count as well.
+            if epv['pkg'].get('cooccurrence_count'):
```

Comment:
`pkg_dict['cooccurrence_count'] = epv['pkg'].get('cooccurrence_count', 0)` 

This is much better in context of usage in `recommender.py` Can you change it? @sara-02

Reply:
sure :-) I completely understand, but still, would you like to open new PR?

Reply:
@pkajaba Sure, will add a new PR and  fix the issue in other places in this file.